### ISSUE TITLE
Replace \bar with \overline and \pi with π etc

### ISSUE THREAD

**Issue by HDembinski** [scikit-hep/particle#289]
The LaTeX strings for the particles are very nice and very useful, but they could be improved.
```py
from particle import literals as lp
lp.pi_plus.latex_name # '\\pi^{+}'
lp.Lambda_bar.latex_name # '\\bar{\\Lambda}'
```

1) Use upright greek letters. It is CERN wide style to use upright greek letters instead of slanted. The LHCb style imports the `upgreek` package for this, which we cannot do here, but we can replace `\\pi^{+}` with `π^{+}` to get the same effect. Likewise for other particles.

2) Use `\overline` instead of `\bar`. The LHCb style also uses `\overline` instead of `\bar`, since the latter does not look very good on wide characters like the Λ.

**Comment by HDembinski** [comment]
tl;dr: I retract one of the two suggested changes, we should only replace \bar by \overline.

First mistake: replacing $\pi$ with π does not give the desired effect, you have to use \mathrm{π}, too. But I am not sure anymore whether we actually want that. I did some further research and it seems that particle names should be italic.

The LHCb style says so.
https://gitlab.cern.ch/lhcb-docs/templates/-/blob/master/LHCb-latex-template/latest/latex/body.tex#L287

And here is a recent paper from ATLAS that also uses italic particle names.
https://inspirehep.net/files/656faee2a0f9df70f1bdb590645dc299

I got confused by the implementation here:
https://gitlab.cern.ch/lhcb-docs/templates/-/blob/master/LHCb-latex-template/latest/latex/lhcb-symbols-def.tex
which allows using upright letters, but it is optional.

**Comment by eduardo-rodrigues** [comment]
Thanks for your suggestion, @HDembinski. I believe you are right about point 2 but would also be in principle against your suggestion 1, though I will make some checks on my side. Indeed we should not impose LHCb styles just because we are LHCb, and should try and make the overall community happy.

I will make a PR as soon as I check a few bits of code that your suggestions made me think of.

**Comment by eduardo-rodrigues** [comment]
Further on the discussion of unicodes in particle names, @HDembinski:
in fact I had a long time ago played with unicodes and had left in `particle.utilities` some code that makes use of the `unicodedata` package to provide `Omega -> Ω` and alike. So, if I simply add there a little utility function
```
def latex_name_unicode(name):
    # type: (str) -> str
    """
    Convert in particle names in LaTeX all greek letters by their unicode.

    Examples
    --------
    >>> n = Particle.from_pdgid(3124).latex_name
    >>> n
    '\\Lambda(1520)'
    >>> latex_name_unicode(n)
    'Λ(1520)'
    """
    if "ambda" in name:
        name = name.replace("ambda","amda")  # Unicodedata library uses "lamda" for "lambda" :S!
    for gl in _list_name_greek_letters:
        name = name.replace(r"\%s" % gl, "%s" % greek_letter_name_to_unicode(gl))
    return name
```
this does what you were talking about, on the fly. That's something easy to add for the user. Is that something you would welcome? I am tempted to add it to https://github.com/scikit-hep/particle/pull/290 while at it.
